user/waydroid/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "waydroid"
+pkgver = "1.4.3"
+pkgrel = 0
+build_style = "makefile"
+make_build_args = ["USE_SYSTEMD=0", "USE_NFTABLES=1"]
+depends = [
+    "dbus",
+    "dnsmasq",
+    "lxc",
+    "python",
+    "python-dbus",
+    "python-gbinder",
+    "python-gobject",
+    "python-pyclip",
+]
+pkgdesc = "Container-based approach to boot a full Android system"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/waydroid/waydroid"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "6557c6fed6a0a7417503eaaab3602efd67c6ced2026725ac24ec8c809fc672e4"
+# check: no tests
+options = ["!check"]
+
+
+def post_install(self):
+    self.uninstall("usr/lib/systemd*", glob=True)
+    self.install_service(self.files_path / "waydroid-container")
+    self.install_file(
+        self.files_path / "51_waydroid.nft",
+        "etc/nftables.d",
+        name="51_waydroid.nft",
+    )
+
+
+@subpackage("waydroid-nftables")
+def _(self):
+    self.depends = ["nftables"]
+    self.subdesc = ""
+    self.install_if = [self.parent, "nftables"]
+    return ["etc/nftables.d/51_waydroid.nft"]
```

Comment:
considering this is like a 4kb config file, is there a need to subpackage it at all?

Reply:
i just saw alpine and they had split those. probably would be better to just be into the base instead

Reply:
in alpine they subpackage everything even when it kinda doesn't make sense, in this case it was added like that because of a start to add nft rules for various things that a person wanted (to help with showing how to expose various services), so overall it might make sense one way or another there but here you might as well just throw in the whole file tbh